### PR TITLE
Create a Serverless Logo Generator Application

### PR DESCRIPTION
**Pull request by rbgo404**
PR for the cookbook "Create a Serverless Logo Generator Application"

### CODE REVIEW

**Comment by rbgo404** [comment]
@AGInfer  Please check

**Comment by AGInfer** [comment]
@nickaggarwal : can you check code once before i make it live

File: cookbook/serverless-logo-generator.mdx [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+Here's the custom runtime for the application:
+```yaml
+build:
+  cuda_version: "12.1.1"
+  system_packages:
+    - "libssl-dev"
+    - "libx11-6"
+    - "libxext6"
+    - "libgl1-mesa-glx"
+    - "ffmpeg"
```
Comment (nickaggarwal) [inline cookbook/serverless-logo-generator.mdx:108]: Is ffmpeg requited ?

File: cookbook/serverless-logo-generator.mdx [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+Post-processing idle time before scaling down: (60 seconds - 5.32 seconds) x 100 = 5468 seconds (or 1.52 hours approximately)
+3. __Cold Start Overhead:__<br />
+Total: 100 x 11.72 = 1172 seconds (or 0.33 hours approximately)<br />
+
+__Total Billable Hours with Inferless:__ 14.78 (inference duration) + 1.52 (idle time) + 0.33 (cold start overhead)  = 16.63 hours<br />
+__Total Billable Hours with Inferless:__ 16.63 hours
+
+### Pricing Comparison for all the Scenario
+
+| Scenarios | AWS SageMaker Cost | Inferless Cost |
```
Comment (nickaggarwal) [inline cookbook/serverless-logo-generator.mdx:183]: why not deploy on A10G on inferlesss, why have we taken 100 in inferless ?

File: cookbook/serverless-logo-generator.mdx (hunk 1 above)
Comment (rbgo404) [reply cookbook/serverless-logo-generator.mdx:108]: We can remove the ffmpeg

**Pull request merged**